### PR TITLE
[stdlib] Replace Random.default with SystemRandomNumberGenerator

### PR DESCRIPTION
**Pull request by airspeedswift**
As described in [this forum post](https://forums.swift.org/t/se-0202-amendment-proposal-rename-random-to-defaultrandomnumbergenerator/12942/98) as an amendment to SE-0202.

### CODE REVIEW

**Review by stephentyrone — APPROVED** [review]
I just threw up in my mouth a little bit.

**Comment by airspeedswift** [comment]
cc @Azoy

**Comment by airspeedswift** [comment]
@stephentyrone we aim to please

**Comment by airspeedswift** [comment]
@swift-ci please test

**Review by Azoy — APPROVED** [review]

File: stdlib/public/core/Random.swift [hunk 1]
```diff
@@ -113,38 +113,33 @@ extension RandomNumberGenerator {
 /// The default source of random data.
 ///
 /// When you generate random values, shuffle a collection, or perform another
-/// operation that depends on random data, this type's `default` property is
-/// the generator used by default. For example, the two method calls in this
-/// example are equivalent:
+/// operation that depends on random data, this type's `default` property is the
```
Comment (Azoy) [inline stdlib/public/core/Random.swift]: Some left over docs about the `default` property.

**Review by lorentey — APPROVED** [review]
👍

**Pull request merged**